hosts

```diff
@@ -1,2 +1 @@
-54.186.238.205
-54.68.89.235
\ No newline at end of file
+jarvis.cse.buffalo.edu
```

Comment:
This shouldn't be checked into git. It would be better to define hosts in `config.py` as mentioned below.